WalletWasabi/Blockchain/TransactionBuilding/BranchAndBound.cs

```diff
@@ -11,6 +11,7 @@ namespace WalletWasabi.Blockchain.TransactionBuilding;
 public class BranchAndBound
 {
 	private readonly Random _random = new();
+	private readonly long _defaultTolerance = 500;
 
 	/// <param name="values">All values must be strictly positive.</param>
 	public BranchAndBound(List<long> values)
```

Comment:
This is throwing exception "List is empty" when I try the following:

1. Create new wallet
2. Receive some bitcoin
3. Try to spend unconfirmed UTXO

Error details:

```
System.ArgumentException
  HResult=0x80070057
  Message=List is empty.
  Source=WalletWasabi
  StackTrace:
   at WalletWasabi.Blockchain.TransactionBuilding.BranchAndBound..ctor(List`1 values) in WalletWasabi\Blockchain\TransactionBuilding\BranchAndBound.cs:line 24
   at WalletWasabi.Blockchain.TransactionBuilding.ChangelessTransactionCoinSelector.TryGetCoins(List`1 availableCoins, FeeRate feeRate, Int64 target, List`1& selectedCoins, CancellationToken cancellationToken) in WalletWasabi\Blockchain\TransactionBuilding\ChangelessTransactionCoinSelector.cs:line 21
   at WalletWasabi.Fluent.ViewModels.Wallets.Send.ChangeAvoidanceSuggestionViewModel.<>c__DisplayClass8_0.<GenerateSuggestionsAsync>b__3() in WalletWasabi.Fluent\ViewModels\Wallets\Send\ChangeAvoidanceSuggestionViewModel.cs:line 107
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
```

https://pastebin.com/raw/0v4x3zG2

Reply:
Is this reproducable on master, too?

Reply:
Nevermind I have the fix for this.